### PR TITLE
Feat/gke wrap v1

### PR DESCRIPTION
**Pull request by dnitsch**
#### 📲 What

implements GKE deployment working v1

#### 🛠 How
		
More in-depth discussion of the change or implementation.

#### 👀 Evidence
		
Screenshots / external resources / links / etc.
Link to documentation updated with changes impacted in the PR
		 
#### 🕵️ How to test

Notes for QA

#### ✅ Acceptance criteria Checklist

- [ ] Code peer reviewed?
- [ ] Documentation has been updated to reflect the changes?
- [ ] Passing all automated tests, including a successful deployment?
- [ ] Passing any exploratory testing?
- [ ] Rebased/merged with latest changes from development and re-tested?
- [ ] Meeting the Coding Standards?

### CODE REVIEW

File: libs/images/k8s-deploy/Dockerfile [hunk 1]
```diff
@@ -74,9 +74,8 @@ RUN curl -fsSL "https://github.com/GoogleCloudPlatform/docker-credential-gcr/rel
 | tee /usr/local/bin/docker-credential-gcr > /dev/null && chmod 0777 /usr/local/bin/docker-credential-gcr
 
 # ENVSUBST with built in checks
-RUN curl -fsSL https://github.com/a8m/envsubst/releases/download/v1.1.0/envsubst-`uname -s`-`uname -m` -o /opt/envsubst && \
+# RUN curl -fsSL https://github.com/a8m/envsubst/releases/download/v1.1.0/envsubst-`uname -s`-`uname -m` -o /opt/envsubst && \
```
Comment (estherthetesteramido) [inline libs/images/k8s-deploy/Dockerfile:77]: Remove

File: packages/pact-config/package.json [hunk 2]
```diff
@@ -7,7 +7,7 @@
   "types": "./dist/index.d.ts",
   "scripts": {
     "build": "tsc",
-    "test": "node_modules/.bin/jest --runInBand",
+    "test": "env PACT_CONSUMER=${PACT_CONSUMER:=GenericMenuConsumer} env PACT_PROVIDER=${PACT_PROVIDER:=GenericAPI} node_modules/.bin/jest --runInBand",
```
Comment (estherthetesteramido) [inline packages/pact-config/package.json:10]: Shouldn't need this now

File: packages/pact-config/pact/pacts/genericmenuconsumer-genericapi.json [hunk 3]
```diff
@@ -0,0 +1,43 @@
+{
```
Comment (estherthetesteramido) [inline packages/pact-config/pact/pacts/genericmenuconsumer-genericapi.json:1]: This should get overriden by the tests. They are setting the env variables in scope.

File: packages/scaffolding-cli/templates/deploy/k8s/app/base_gke-app-deploy.yml [hunk 4]
```diff
@@ -74,9 +85,11 @@ spec:
               value: '${k8s_app_path}'
             - name: APP_BASE_URL
               value: 'https://${dns_pointer}'
-
             - name: APPINSIGHTS_INSTRUMENTATIONKEY
-              value: f7c7d8ef-1421-41ce-9c7b-30205ef66ba6
```
Comment (estherthetesteramido) [inline packages/scaffolding-cli/templates/deploy/k8s/app/base_gke-app-deploy.yml:79]: Oops. Do we need to rotate?

Comment (dnitsch) [reply packages/scaffolding-cli/templates/deploy/k8s/app/base_gke-app-deploy.yml:79]: that one was decommissioned already.

**Pull request merged**